pkgs/top-level/darwin-packages.nix

```diff
@@ -148,6 +148,8 @@ impure-cmds // appleSourcePackages // chooseLibs // {
   # Removes propagated packages from the stdenv, so those packages can be built without depending upon themselves.
   bootstrapStdenv = mkBootstrapStdenv pkgs.stdenv;
 
+  libSystem = callPackage ../os-specific/darwin/libSystem { };
```

Comment:
If `stdenv.cc.libc` is `null`, it’s treated as not having a libc by cc-wrapper (`noLibc` must be true). I don’t know what the implication of that is, but Darwin does have a libc. It’s just that it’s found dynamically at build time instead of being known at evaluation time.